evcouplings/visualize/pairs.py

```diff
@@ -1183,43 +1182,101 @@ def enrichment_pymol_script(enrichment_table, output_file,
 
-    # compute boundaries for highly coupled residues
-    # that will be specially highlighted
-    boundary1 = int(0.05 * len(t))  # top 5%
-    boundary2 = int(0.15 * len(t))  # top 15%
+        # high
+        t.loc[t.iloc[0:boundary1].index, "color"] = "red"
 
-    t.loc[:, "b_factor"] = t.enrichment
+        # medium
+        t.loc[t.iloc[boundary1:boundary2].index, "color"] = "orange"
+        
+    else:
+        t = deepcopy(enrichment_table)
+        t.loc[:, "b_factor"] = t.enrichment
+
+        # set boundaries for enrichment levels
+        # that will be specially highlighted
+        # create nine subsets
+        boundary_list = [
+                int(0.11 * len(t)),
+                int(0.22 * len(t)),
+                int(0.33 * len(t)),
+                int(0.44 * len(t)),
+                int(0.55 * len(t)),
+                int(0.66 * len(t)),
+                int(0.77 * len(t)),
+                int(0.88 * len(t)),
+                int(1.00 * len(t))
+            ]
+
+        # list of colors to color each category
+        # must be same length and order as boundary_list
+        # list of rgb tuples
+        color_list = [
+            (77, 0, 75),  # dark purple
+            (129, 15, 124),
+            (136, 65, 157),
+            (140, 107, 177),
+            (140, 150, 198),
+            (158, 188, 218),
+            (191, 211, 230),
+            (224, 236, 244),
+            (247, 252, 253)  # almost white
+        ]
 
-    # set color for "low" enrichment (anything > 1)
-    t.loc[:, "color"] = "yelloworange"
+        # convert to fractions
+        color_list = [(x/255., y/255., z/255.) for x, y, z in color_list]
 
-    # high
-    t.loc[t.iloc[0:boundary1].index, "color"] = "red"
+        prior_boundary = 0
 
-    # medium
-    t.loc[t.iloc[boundary1:boundary2].index, "color"] = "orange"
+        for idx,boundary in enumerate(boundary_list):
+            t.loc[t.iloc[prior_boundary:boundary].index, "color"] = 'color{}'.format(idx)
+            prior_boundary = boundary
 
     if sphere_view:
-        t.loc[t.iloc[0:boundary2].index, "show"] = "spheres"
+        t.loc[t.iloc[0:boundary].index, "show"] = "spheres"
 
     if chain is not None:
         chain_sel = ", chain '{}'".format(chain)
     else:
         chain_sel = ""
 
     with open(output_file, "w") as f:
-        f.write("as cartoon{}\n".format(chain_sel))
-        f.write("color grey80{}\n".format(chain_sel))
 
         if chain is None:
             f.write("alter all, b=0.0\n")
         else:
             f.write("alter chain '{}', b=0.0\n".format(chain))
 
+        # for leagcy mode, background color is grey80
+        if legacy:
+            f.write("color grey80{}\n".format(chain_sel))
+
+        # for non-legacy mode, background color is the last color in the spectrum
+        else:
+            for idx,c in enumerate(color_list):
+                f.write('set_color color{}, [{},{},{}]\n'.format(idx, c[0], c[1], c[2]))
```

Comment:
" quotes instead of '

(maybe useful info: in pymol one can directly set colors using hexademical colors codes prefixed by 0x instead of #, e.g. color 0xff0000 - took me a while to discover this)